perfkitbenchmarker/alicloud/util.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2015 Google Inc. All rights reserved.
```

Comment:
Please update this from "Google Inc" to "PerfKitBenchmarker Authors".
Also add "Alibaba Inc" or if more appropriate "Alicloud Inc" or if this is your personal change your name to the AUTHORS file.